Orange/widgets/utils/tests/test_state_summary.py

```diff
@@ -183,6 +184,15 @@ def test_details(self):
                   f'Target: —'
         self.assertEqual(details, format_summary_details(data))
 
+        data = Table.from_numpy(domain=None, X=np.random.random((10000, 1000)))
+        details = f'{len(data):n} instances, ' \
+                  f'{len(data.domain.variables)} variables\n' \
+                  f'Features: {len(data.domain.variables)} numeric \n' \
+                  f'Target: —'
+        with patch.object(Table, "get_nan_frequency_attribute") as mock:
+            self.assertEqual(details, format_summary_details(data))
```

Comment:
I should have the same objection as above. But these tests are already written the wrong (imho) way, so ... let it stay.